pkg/transfers/inbound/cleanup.go

```diff
@@ -25,13 +28,62 @@ func Cleanup(logger log.Logger, agent upload.Agent, dl *downloadedFiles) error {
 	if err := deleteFilesOnRemote(logger, agent, dl.dir, agent.ReturnPath()); err != nil {
 		el.Add(err)
 	}
+	if el.Empty() {
+		return nil
+	}
+	return el
+}
+
+// DeleteZeroByteFiles deletes empty ACH files if file older than value in config
+func DeleteZeroByteFiles(logger log.Logger, agent upload.Agent, dl *downloadedFiles, after time.Duration) error {
```

Comment:
Nit: I think we want to `return nil` early if `after <= 0*time.Second`. That would imply two things.

1. By default we're not deleting remote files which allows in-progress uploads to complete
   - Also requires the operator deploying paygate to set a non-zero value if they want such files deleted
1. Allowing this to be disabled (by default) or with a negative value

Reply:
Agreed. Pushed.